home/ivp-planning/machines/first-steps-with-3d-object.md

```diff
@@ -24,8 +24,10 @@ If you selected the wrong object, you can change your selection by clicking on t
 
 ![](../../../.gitbook/assets/iVP\_guide\_add\_new\_machines\_selected\_machine.jpg)
 
-The last step is to simply click anywhere in the 2D- or the [3D-panel](../user-interface/the-3d-panel.md). The previously selected machine will now appear in the scene in the spot you clicked and the preview window will close automatically.
+The last step is to simply click anywhere in the [2D-panel](
+../user-interface/the-2d-panel.md
+) or the [3D-panel](../user-interface/the-3d-panel.md). The previously selected machine will now appear in the scene in the spot you clicked and the preview window will close automatically.
```

Comment:
```suggestion
The last step is to simply click anywhere in the [2D-panel](../user-interface/the-2d-panel.md) or the [3D-panel](../user-interface/the-3d-panel.md). The previously selected machine will now appear in the scene at the spot you clicked and the preview window will close automatically.
```

There shouldn't be  a line break here and use _at the spot_ instead of _in the spot_.